.github/workflows/ci.yml

```diff
@@ -8,31 +8,25 @@ env:
 
 jobs:
   test-sqlite:
-    strategy:
-      fail-fast: false
-      matrix:
-        combination: ['v5', 'v6', 'v6 with TS']
-    name: SQLite (${{ matrix.combination }})
+    name: SQLite (sequelize 6)
     runs-on: ubuntu-latest
     env:
       DIALECT: sqlite
-      CI_COMBINATION: ${{ matrix.combination }}
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 12.x
+          node-version: 14.x
```

Comment:
Why not 16 or the current LTS?

Reply:
I was initially going to leave it as 12, because we support up to node 12, but ESM support in node 12 is broken (specifically the loader API), so I moved to node 14

I think it's better to revert the move to ESM for now, and use node 12. At least until next october.

Reply:
Soo, it's been almost two months and I forgot about this PR

I propose to do this:
- run Sequelize 6 against node 10
- run Sequelize 7 against node 14

As they're the the minimum supported versions

Really hoping we'll be able to deprecate Sequelize 6 soon so we don't have to keep supporting dead node versions 😬

Reply:
Sounds good, if people have issues with later Node versions they can always update the CI file as well